### PR TITLE
Add note to changelog about changing dependencies

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@ Change Log
 unreleased
 ~~~~~~~~~~
 
+* ``attrs`` is no longer a dependency; ``ast_decompiler`` has been
+  added as a dependency
```
Comment (Akuli) [inline CHANGELOG.rst:8]:
```suggestion
  added as a dependency on Python 3.8 and older
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,8 @@ Change Log
 unreleased
 ~~~~~~~~~~
 
+* ``attrs`` is no longer a dependency; ``ast_decompiler`` has been
+  added as a dependency on Python 3.8 and 3.7
```
Comment (JelleZijlstra) [inline CHANGELOG.rst:8]:
```suggestion
* ``attrs`` is no longer a dependency
* ``ast_decompiler`` has been
  added as a dependency on Python 3.8 and 3.7
```
They're two unrelated changes

**Review by JelleZijlstra — APPROVED** [review]
